hono/src/drizzle/0002_tired_firelord.sql

```diff
@@ -0,0 +1,2 @@
+-- Custom SQL migration file, put you code below! --
+CREATE MATERIALIZED VIEW "items_extended" AS SELECT * FROM "items";
```

Comment:
Should this be `CREATE MATERIALIZED VIEW IF NOT EXISTS ...`?

Reply:
Good nit. Will implement if we decide to stick with the materialized view: https://github.com/radicalplatforms/shafa/pull/104#discussion_r1513571522

Reply:
Sounds good! I'd leave a thumbs up but GitHub won't let me use an emoji on this message only for some reason.